components/prism-scheme.js

```diff
@@ -17,17 +17,18 @@ Prism.languages.scheme = {
 		lookbehind: true
 	},
 	'number': {
-		pattern: /(\s|[()])[-+]?\d*\.?\d+(?:\s*[-+]\s*\d*\.?\d+i)?\b/,
+		newPattern: /(?:#d(?:#[ei])?|#[ei](?:#d)?)?/i,
```

Comment:
Is `newPattern` a thing? And should there be associated tests for this?

Reply:
Ahhh, big mistake on my side. I sometimes test out new patterns like this.
No, `newPattern` is not a thing.